### ISSUE TITLE
Error "Dynamic operations can only be performed in homogenous AppDomain"

### ISSUE THREAD

**Issue by pwallner** [adoconnection/SevenZipExtractor#36]
With version 1.0.13 (not with 1.0.12), I got this error when used in conjuction with this app.config setting:
```
<NetFx40_LegacySecurityPolicy` enabled="true" />
```
The full error I get is:
```
System.InvalidOperationException: Dynamic operations can only be performed in homogenous AppDomain.
```
If I remove the app.config setting, everything works great. Is there a chance to make these working?

Thanks!

**Comment by adoconnection** [comment]
@pwallner published nuget 1.0.14, now should work fine for NetFx40_LegacySecurityPolicy=true

**Comment by pwallner** [comment]
Thank you very much!
